edge-hub/docker/linux/amd64/Dockerfile

```diff
@@ -18,18 +21,23 @@ RUN adduser -Ds /bin/sh -u ${EDGEHUBUSER_ID} edgehubuser
 RUN setcap 'cap_net_bind_service=+ep' /usr/share/dotnet/dotnet	
 
 # Install RocksDB	
-COPY --from=builder publish/* /usr/local/lib/	
+COPY --from=hub-builder publish/* /usr/local/lib/	
 
 WORKDIR /app	
 
-COPY $EXE_DIR/ ./	
+# Copy edge hub artifacts and remove unnecessary items, leaving just edge hub core dlls
+COPY $EXE_DIR/ ./
```

Comment:
We need to remove docker files, mqtt binary, watchdog binary. I tried adding a rm step, but since the builder agent is amd64 this won't work on arm. Any ideas?